packages/block-library/src/paragraph/index.js

```diff
@@ -28,7 +28,17 @@ export const settings = {
 		},
 	},
 	__experimentalLabel( attributes, { context } ) {
+		const customName = attributes?.metadata?.name;
+
+		if ( context === 'list-view' && customName ) {
```

Comment:
Is this something we only need for `list view`? I'm wondering if it should be part of helper hooks like `useBlockDisplayInformation` or `useBlockDisplayTitle`, or even `getBlockLabel` function etc..

There might be nuances I'm missing about why/where we use `__experimentalLabel`.

Reply:
For now the output of the custom name is only utilised in the List View.

If in future there is a use case for utilising the custom name _outside_ of List View then what you recommend would be ideal 👍